README.md

```diff
@@ -6,56 +6,55 @@ TSMP2 workflow engine for running simulations. The following examples and descri
 
 ## Setup the workflow
 
+Activate a compute project
+```bash
+# Replace PROJECTNAME with your compute project
+jutil env activate -p PROJECTNAME
+
+# Check if $BUDGET_ACCOUNTS was set.
+echo $BUDGET_ACCOUNTS
+```
+
+In case you are not on a [JSC](https://www.fz-juelich.de/) machine, set the shell variables `BUDGET_ACCOUNT`, `PROJECT` and `SCRATCH` manually.
+Instead of setting `BUDGET_ACCOUNT` you may also replace this variable in `ctl/control_tsmp2.sh`
+
 ``` bash
-cd $PROJECT_DIR
-git clone --recurse-submodules https://github.com/HPSCTerrSys/TSMP2_workflow-engine
-wfe_dir=realpath tsmp2_workflow-engine
+cd $PROJECT/$USER
+git clone https://github.com/HPSCTerrSys/TSMP2_workflow-engine
+wfe_dir=$(realpath tsmp2_workflow-engine)
+git submodule init
+git submodule update
 ```
 
 ## Building the model
 
 The TSMP2 ( https://github.com/HPSCTerrSys/TSMP2 ) should be either already compiled (see [ReadMe TSMP2](https://github.com/HPSCTerrSys/TSMP2/blob/master/README.md)) or compiled with the following steps.
 
-0) change directory
 ```bash
 cd ${wfe_dir}/src/TSMP2
+./build_tsmp2.sh --tsmp2_env env/jsc.2024_Intel.sh --icon --eclm --parflow
```

Comment:
Please take the default environment in the ReadMe and not mention it explicitly.

Reply:
Fixed in 8a31a9b0f46cc663f46b756354db7f0d5170538a.